flake.nix

```
@@ -54,8 +54,8 @@
           (pkgs.haskellPackages.callCabal2nix "hevm" (pkgs.fetchFromGitHub {
             owner = "ethereum";
             repo = "hevm";
-            rev = "9982c580ed19b88ebab9744d29d940fd2f0bd8c6";
-            sha256 = "sha256-B60aJo9EfIaTWuJaN06Wl1Br+rstzUAYGFBE5rX72Fs=";
+            rev = "87fe0eec5abb69c0b54c097784dfd8712a36de7";
```

```suggestion
            rev = "87fe0eec5abb69c0b54c097784dfd8712a36de70";
```

let's use full commit hashes 😄